policy/observers.md

```diff
@@ -0,0 +1,22 @@
+# Observers
+
+**Administrator:** A Working Group in Electron Governance or a company represented in a Working Group wishing to introduce an Observer into their Working Group.
+
+**Temporary worker:** A person employed or otherwise contracted by an Administrator for a temporary and fixed duration. Their role must entail that they primarily work on the Electron framework and/or other responsibilities of the Working Group they wish to join.
```

Comment:
Suggestions: 
* Observer (what the interns used last year) 
<img width="662" alt="Screen Shot 2020-08-24 at 3 53 23 PM" src="https://user-images.githubusercontent.com/2138661/91104143-08415f80-e622-11ea-94f9-47e506e6a811.png">

* Collaborator (see definition https://github.com/electron/governance#definitions)

Reply:
What about "Temporary collaborator"? I'm hesitant to use either of those directly because there should be a distinction between the role they need to qualify for Observership and the Observer role itself. Likewise, the existing collaborator definition doesn't really encompass what we need here.

Reply:
Great point. I'd be good with "Temporary Collaborator"